sdk/textanalytics/tests.yml

```diff
@@ -12,5 +15,6 @@ stages:
       # Will enable it when service rolls out AAD to all region.
       DisableAzureResourceCreation: true
       EnvVars:
+        AZURE_TEST_MODE: RECORD
```

Comment:
You only need to set `TestMode`, setting both `TestMode` and `AZURE_TEST_MODE` will result in an invalid YML configuration